lambdas/data-migration2/package.json

```diff
@@ -11,7 +11,7 @@
     "clean": "rm -rf dist",
     "build-lambda-zip": "cd dist/webpack && node ../../../../bin/zip.js lambda.zip index.js",
     "package": "npm run clean && npm run prepare && npm run webpack && npm run build-lambda-zip",
-    "test": "../../node_modules/.bin/ava",
+    "test": "../../node_modules/.bin/ava --serial",
```

Comment:
I would prefer not to set this here in the package config and instead just add `.serial` to every test where we need it

Reply:
Hm, I am willing to remove this but doing so doesn't achieve the same thing.

Reply:
Fair enough, but if we need it then we have some unexpected interference among our tests and I'd like to understand why